### ISSUE TITLE
Turbo and N/A engines with new Boost/blowOffVentilSound?

### ISSUE THREAD

**Issue by MrMazzony** [Mogli12/GearboxAddon#432]
Hi.
Is there way to use "blowOffVentilSound" in <engine> instead of <transmission>. This way it would be easyer to configure N/A and Turbo engines wirh multiple transmission options.
Or am i forced to use fullBoostMs in engine instead?

**Comment by Mogli12** [comment]
Hi,
With <blowOffVentilSound volume=0/> you will turn off the boost thing.
Regards, Stefan

**Comment by MrMazzony** [comment]
Yes, but isnt that in <transmissions> tag? And can you use that in <engines> tag?

**Comment by Mogli12** [comment]
```xml
<transmissions>
  <transmission name="No 1" ...>
    <blowOffVentilSound volume=0/>
    ...
  </transmission>

  <transmission name="No 2" ...>
    <blowOffVentilSound volume=0/>
    ...
  </transmission>
</transmissions>

**Comment by MrMazzony** [comment]
So i DO need to include it in the transmission tags. I wouldnt want to create seperate transmissions for Turbo and N/A engines with same gear layout.
IE: 
Transmission1
Transmission 1 Turbo
😣

**Comment by Dudejo** [comment]
There IS a possible way to do what you want. I don't recall which exact topic but there was a way to tie a specific engine to a specific transmission. It's only one transmission per engine, though.

**Comment by MrMazzony** [comment]
@Dudejo. That way is the "old" way of using "GearboxMogli" tags. That allows many engineconfigs, but not transmission selection posibility. And i need transmission choosability.

**Comment by Mogli12** [comment]
It depends if you want to have the configuration in gearboxAddonConfig.xml or in the XML file of the vehicle; e.g. via moreRealisticXmlDatabank.

In gearboxAddonConfig.xml:
```xml
<?xml version="1.0" encoding="utf-8" standalone="no" ?>
<vehicles>
	<vehicle>
<!-- config for the 1st engine -->
		<mod name="FS17_Fendt_1050">
			<xmlFile name="xml/fendt1050.xml" engines="1"/>
		</mod>
		<gearboxMogli maxBackwardSpeed="33" disableManual="true">
...
		</gearboxMogli>
	</vehicle>
	<vehicle>
<!-- config for the 2nd engine -->
		<mod name="FS17_Fendt_1050">
			<xmlFile name="xml/fendt1050.xml" engines="2"/>
		</mod>
		<gearboxMogli maxBackwardSpeed="33" disableManual="true">
...
		</gearboxMogli>
	</vehicle>
...
</vehicles>
```

In the XML file of the vehicle you can use the gearboxMogli tag underneath the motorConfiguration tag. 

But you cannot use `<transmissions>` for engine specific configuraitons

**Comment by Dudejo** [comment]
@MrMazzony

Sadly, there's no way at this time to have a range of transmissions exclusive to one engine.

The only suggestion I can offer is to create a vehicle mod that acts as a clone of the vehicle you want. You can then make a full range of transmissions for that clone.

**Comment by Mogli12** [comment]
The different configuration options are independent from each other. That is why it is not possible to offer a range of transmissions only for certain motor configurations.

**Comment by MrMazzony** [comment]
This might have derailed abit.
Im not lookin to do transmission options limits per engine (trans 1&2 for engine1/ trans 3&4 for engine2) . No.

I want blowoffVentilSound to be toggled trough engine configuration, not trough transmision selection so i dont need to do different transmissions for turboengine to get the sound/boost right.

**Comment by Dudejo** [comment]
@MrMazzony 
I understand what you mean.

But in the meantime, the only way to do what you want is using the methods described above.

**Comment by MrMazzony** [comment]
@Mogli12
Is that blowOffVentilSound hardcoded to Transmissions only, and could it be moved to Engines instead.
Reaaons:
Engines are the part that has the blowoff/turbo irl, not the transmission. I underatand that sound effext is more dependnt on chancing gears than engine, but it would be more realistic ti use that in engines. 

Regards: MrMazzony
